Prog/LgUnit - Export ACS Tract Data for Maps.sas

```diff
@@ -0,0 +1,59 @@
+/**************************************************************************
+ Program:  LgUnit - Export ACS Tract Data for Maps.sas
+ Library:  DMPED
+ Project:  NeighborhoodInfo DC
+ Author:   Rob Pitingolo
+ Created:  7/6/18
+ Version:  SAS 9.4
+ Environment:  Windows with SAS/Connect
+ 
+ Description:  Export CSV of 2012-16 ACS data for large units maps. 
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( ACS )
+
+%let geo_suffix = _tr10;
+%let geo = geo2010;
+
+
+data ACS_2012_16_map; 
+	set acs.acs_2012_16_dc_sum_tr&geo_suffix.;
+
+	numhsgunits_2012_16 = sum(of numhsgunits0bd_2012_16 numhsgunits1bd_2012_16 numhsgunits2bd_2012_16 numhsgunits3plusbd_2012_16);
+
+	keep &geo. all3brplusrentunits numrnt3br_under1000 numrnt3br_under1500 pctrnt3br_under1000 pctrnt3br_under1500 pct3brrent;
+
+	all3brplusrentunits = sum(of numrtohu3bunder500_2012_16 numrtohu3b500to749_2012_16 numrtohu3b750to999_2012_16 numrtohu3b1000plus_2012_16
```

Comment:
having numrtohu3b1000plus_2012_16 in this sum doesn't look right.

Reply:
@rpitingolo please look at this reviwe

Reply:
@lhendey nice catch. I was double counting $1000+. I've removed the duplicate var.